overrides/scripts/_oreDict.zs

```diff
@@ -2450,6 +2450,18 @@ recipes.addShapeless("of_exchange_neutronium_plate",
 // Workaround for DE Fusion Custom recipe not liking strings in the NBT
 <ore:compressedoctadiccap>.add(compressedoctadiccap);
 
+// OreDict stuff for new Pulsating Mesh/Dust
+oreDict.get("dustPulsating");
+oreDict.get("meshPulsating");

```

Comment:
Are these two calls to retrieve the oreDict objects necessary here? It seems like they are just discarded and you get them with a bracket handler in the subsequent lines.

Reply:
I believe the `get` call will create the oredicts if they do not exist, which I do not think the assignment below will do.

Reply:
Hmm, interesting. I'll check what happens without it.

Reply:
It appears to have worked fine with those lines removed.